ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json

```diff
@@ -416,5 +416,27 @@
       "caseworker-publiclaw-magistrate"
     ],
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftSDO",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-publiclaw-gatekeeper",
+          "caseworker-publiclaw-judiciary"
+        ],
+        "CRUD": "CR"
```

Comment:
Needs to be just R, don't want them to use old event

Reply:
i.e move them into the section down below

Reply:
oh very good point haha!